src/Microsoft.VisualStudio.LanguageServices.Razor/DefaultTagHelperResolver.cs

```diff
@@ -90,5 +105,11 @@ private void AddXmlDocumentation(Compilation compilation, IReadOnlyList<TagHelpe
                 }
             }
         }
+
+        private IVsActivityLog GetActivityLog()
+        {
+            var services = (IServiceProvider)Services;
```

Comment:
Is the cast needed? (Going by https://msdn.microsoft.com/en-us/library/microsoft.visualstudio.shell.svsserviceprovider.aspx, it shouldn't be).